site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -29,7 +29,7 @@
         <div id="upload-boxes">
             {# upload1 and input_file1 required for drag-and-drop.js #}
             <div id="upload1" class="file-input">
-                <label class="label" for="input-file">"Drag your file(s) here or click to open file browser"</label>
+                <label class="label" for="input_file1">"Drag your file(s) here or click to open file browser"</label>
                 <input type="file" name="files" id="input_file1" onchange="addFilesFromInput(1,false)" multiple />
```

Comment:
could you update this to be `kebab-case` here please?

Reply:
done